packages/protocol/test-sol/unit/common/FeeHandler.t.sol

```diff
@@ -102,6 +103,7 @@ contract FeeHandlerTest is Test, Constants {
     registry.setAddressFor("GoldToken", address(celoToken));
     registry.setAddressFor("CeloToken", address(celoToken));
     registry.setAddressFor("Reserve", address(mockReserve));
+    registry.setAddressFor("CeloDistributionSchedule", address(celoDistributionSchedule));
```

Comment:
Nit: `celoDistributionSchedule` is already an `address` from here `address celoDistributionSchedule = actor("CeloDistributionSchedule")`.

```suggestion
    registry.setAddressFor("CeloDistributionSchedule", celoDistributionSchedule);
```